.github/CODEOWNERS

```diff
@@ -0,0 +1,267 @@
+
+Tasks/CMakeV1/*  @daveken
+
+Tasks/CmdLineV2/*   bryanmacfarlane
+
+Tasks/CocoaPodsV0/*     @madhurig
+
+Tasks/CondaEnvironmentV0/*      @madhurig
+
+Tasks/CondaEnvironmentV1/*      @madhurig
+
+Tasks/CopyFilesV2/*     @bryanmacfarlane
+
+Tasks/CopyFilesOverSSHV0/*      @madhurig
+
+Tasks/CUrlUploaderV2/*      @madhurig
+
+Tasks/DecryptFileV1/*   @madhurig
+
+Tasks/DelayV1/*     @bansalaseem
+
+Tasks/DeleteFilesV1/*   @bryanmacfarlane
+
+Tasks/DeployVisualStudioTestAgentV2/*       @smalpani
+
+Tasks/DockerV0/*    @bansalaseem
+
+Tasks/DockerV1/*    @bansalaseem
+
+Tasks/DockerComposeV0/*     @bansalaseem
+
+Tasks/DotNetCoreCLIV2/*     @sachinma
+
+Tasks/DotNetCoreInstallerV0/*   @sachinma
+
+Tasks/DownloadBuildArtifactsV0/*    @Lovakumar      @omeshp
+
+Tasks/DownloadFileshareArtifactsV0/*    @Lovakumar      @omeshp
+
+Tasks/DownloadPackageV0/*       @zackrun
+
+Tasks/DownloadPipelineArtifactV0/*      @mihaif
+
+Tasks/DownloadSecureFileV1/*    @madhurig
+
+Tasks/ExtractFilesV1/*      @bryanmacfarlane
+
+Tasks/FtpUploadV1/*     @madhurig
+
+Tasks/GithubReleaseV0/*     @prativen   @mdmdakbar
+
+Tasks/GoV0/*    @sachinma
+
+Tasks/GoToolV0/*    @sachinma
+
+Tasks/GradleV2/*    @madhurig
+
+Tasks/GruntV0/*     @hross
+
+Tasks/GulpV0/*      @hross
+
+Tasks/GulpV1/*      @hross
+
+Tasks/HelmDeployV0/*    @bansalaseem
+
+Tasks/HelmInstallerV0/*     @bansalaseem
+
+Tasks/IISWebAppDeployment/*     @kmkumaran
+
+Tasks/IISWebAppDeploymentOnMachineGroupV0/*     @kmkumaran
+
+Tasks/IISWebAppManagementOnMachineGroupV0/*     @kmkumaran
+
+Tasks/InstallAppleCertificateV2/*   @madhurig
+
+Tasks/InstallAppleProvisioningProfileV1/*   @madhurig
+
+Tasks/InstallSSHKeyV0/*     @madhurig
+
+Tasks/InvokeRestApiV1/*     @bansalaseem
+
+Tasks/JavaToolInstallerV0/*     @madhurig
+
+Tasks/JenkinsDownloadArtifactsV1/*      @Lovakumar      @kasubram
+
+Tasks/JenkinsQueueJobV2/*   @madhurig
+
+Tasks/KubernetesV0/*    @bansalaseem
+
+Tasks/KubernetesV1/*    @bansalaseem
+
+Tasks/ManualInterventionV8/*
+
+Tasks/MavenV2/*     @madhurig
+
+Tasks/MavenV3/*     @madhurig
+
+Tasks/MSBuildV1/*   @daveken
+
+Tasks/MysqlDeploymentOnMachineGroupV1/*     @kmkumaran      @RoshanKumarMicrosoft
+
+Tasks/NodeToolV0/*      @hross
+
+Tasks/NpmV1/*       @zackrun
+
+Tasks/NpmAuthenticateV0/*       @zackrun
+
+Tasks/NuGetV0/*     @zackrun
+
+Tasks/NuGetCommandV2/*      @zackrun
+
+Tasks/NuGetPackagerV0/*     @zackrun
+
+Tasks/NuGetPublisherV0/*    @zackrun
+
+Tasks/NuGetToolInstallerV0/*    @zackrun
+
+Tasks/PackerBuildV0/*   @sachinma
+
+Tasks/PackerBuildV1/*   @sachinma
+
+Tasks/PowerShellV2/*    @bryanmacfarlane
+
+Tasks/PowerShellOnTargetMachinesV1/*    @kmkumaran
+
+Tasks/PowerShellOnTargetMachinesV2/*    @kmkumaran
+
+Tasks/PowerShellOnTargetMachinesV3/*    @kmkumaran
+
+Tasks/PublishBuildArtifactsV1/*     @madhurig
+
+Tasks/PublishCodeCoverageResultsV1/*    @madhurig
+
+Tasks/PublishPipelineArtifactV0/*       @mihaif
+
+Tasks/PublishSymbolsV2/*    @smalpani
```

Comment:
PublishSymbols assignment is incorrect.

Reply:
who is the task owner?

Reply:
@arunkm - I am marking you as the owner based on history in commits. Please feel free to call out if you are not the owner.